CONTRIBUTING.md

```diff
@@ -104,7 +104,7 @@ func (c ExampleOperatorConfig) Build(context operator.BuildContext) (operator.Op
 
 ```go
 func init() {
-	operator.Register("example_operator", &ExampleOperatorConfig{})
+	operator.RegisterOperator("example_operator", &ExampleOperatorConfig{})
```

Comment:
Since this function is in the operator package, `Register` seems sufficient. I believe this qualifies as stutter according to golang norms.

Reply:
I think this occurrence still exists (though it's only a doc issue).